src/internal/connector/exchange/service_iterators.go

```diff
@@ -349,19 +327,17 @@ func FetchContactIDsFromDirectory(
 			ids = append(ids, *item.GetId())
 		}
 
-		addtlData := resp.GetAdditionalData()
-
-		delta := getAdditionalDataString(deltaLinkKey, addtlData)
-		if len(delta) > 0 {
-			deltaToken = delta
+		delta := resp.GetOdataDeltaLink()
+		if delta != nil {
```

Comment:
nit: could you also augment the checks like this to ensure it's not an empty string
```suggestion
		if delta != nil && len(*delta) > 0 {
```

Reply:
Resolved with [f52cf5b](https://github.com/alcionai/corso/pull/1745/commits/f52cf5baae7b45ff44a6aafdca24fb6d8f9b5de4)